site/docs-md/guides/community.md

```diff
@@ -0,0 +1,17 @@
+# Community Guides
+
+## General
+
+[Capacitor: Five Apps in Five Minutes](https://www.youtube.com/watch?v=x2jRs7Qqai8)
+
+[Create an Ionic 4 PWA with Capacitor](https://blog.jscrambler.com/create-an-ionic-4-pwa-with-capacitor/)
+
+## Integrations
+
+[Using Google Maps and Geolocation in Ionic with Capacitor](https://www.joshmorony.com/using-google-maps-and-geolocation-in-ionic-with-capacitor)
+
+[Create Native HTML5 Games with Phaser and Capacitor](https://www.joshmorony.com/create-native-html5-games-with-phaser-and-capacitor)
+
+[Integrating the Scanbot SDK in a Capacitor App with Ionic](https://scanbot.io/blog/capacitor-app/)
+
+[Sending Push Notifications with the PhoneGap Push Plugin in Capacitor](https://www.joshmorony.com/sending-push-notifications-with-the-phonegap-push-plugin-in-capacitor/)
```

Comment:
This one probably doesn't work anymore, so I wouldn't link it

Reply:
removed